mmv1/templates/terraform/examples/compute-reservation.tf.erb

```diff
@@ -0,0 +1,20 @@
+# [START compute_reservation_create_local_reservation]
```

Comment:
Sorry! I should have mentioned, along with the `name` change, the name of this file will need to change to `compute_reservation`. Sorry about that!

Reply:
Made the changes again